src/mantine-core/src/Grid/Col/Col.styles.ts

```diff
@@ -30,8 +32,29 @@ interface ColStyles {
   orderXl: React.CSSProperties['order'];
 }
 
-const getColumnWidth = (colSpan: number, columns: number) =>
-  colSpan ? `${100 / (columns / colSpan)}%` : undefined;
+const getColumnFlexBasis = (colSpan: ColSpan, columns: number) => {
+  if (colSpan === 'content') {
+    return 'auto';
+  }
```

Comment:
Rather than using flex basis `content`, we're using flex basis `auto` together with a width of `auto` to support older browsers

![image](https://user-images.githubusercontent.com/6363419/187939520-52a973ba-310a-4cdc-828e-2b28e50058c5.png)